src/Microsoft.DncEng.SecretManager/SecretTypes/PgpKeyPair.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.DncEng.CommandLineLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DncEng.SecretManager.SecretTypes
+{
+    [Name("pgp-key-pair")]
+    internal class PgpKeyPair : SecretType<PgpKeyPair.Parameters>
+    {
+        private readonly IConsole _console;
+
+        public class Parameters { }
+    
+        public PgpKeyPair(IConsole console)
+        {
+            _console = console;
+        }
+
+        protected override async Task<SecretData> RotateValue(Parameters parameters, RotationContext context, CancellationToken cancellationToken)
+        {
+            // Adding this to suppress the warning about not having the 'await' operator in an async method
+            await context.GetSecretValue(new SecretReference(context.SecretName));
+
+            throw new HumanInterventionRequiredException($"Pgp key pair secret rotation required. Human intervention required. Please go to https://aka.ms/signalr-java-publishing and follow the instructions at the bottom of the page");
```

Comment:
Shouldn't there be more code here? I would expect that after the help message is thrown, a prompt asking for the new values will show up. You can check an example here: https://github.com/dotnet/arcade-services/blob/main/src/Microsoft.DncEng.SecretManager/SecretTypes/TextSecretType.cs

Reply:
This is not how this is supposed to work. This exception should be thrown if the console isn't interactive.

If the console is interactive this needs to prompt for new values.